types/index.d.ts

```diff
@@ -0,0 +1,12 @@
+import { FastifyMiddleware } from 'fastify'
```

Comment:
This will have to be changed to support fastify v3 types after that is released

Reply:
ok, but it will be a new (major) release of for the plugin, so let's see later (but we need to remember) ... thanks for now